index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  	<title>Happy Trees</title>
+    <link rel="stylesheet" type="text/css" href="CSS/styles.css">
+  </head>
+  <body>
+  <nav>
+      <a href="aboutme.html">About Me</a> <a href="portfolio.html">Portfolio</a>
+    </nav>
+
+
+  <img src="Images/bobross.jpg">
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load. 🔨